library/src/G4CMPEmpiricalLindhardNIEL.cc

```diff
@@ -0,0 +1,115 @@
+/***********************************************************************\
+ * This software is licensed under the terms of the GNU General Public *
+ * License version 3 or later. See G4CMP/LICENSE for the full license. *
+\***********************************************************************/
+
+/// \file library/include/G4CMPEmpiricalLindhardNIEL.cc
+/// \brief NIEL calculation from Ionization yield measurement 
+/// in a germanium CDMSlite detector using photo-neutron sources.
+///
+/// Computation of NIEL according to Lindhard (Lewin Smith) yield with 
+/// a parameter k that depends on the energy.
```

Comment:
See my comment in the .hh file, and provide the same description of how the function works (what does the user specify, and how are those specified values used).